### PR TITLE
Some suggestions for the Principles documention 

### PR DESCRIPTION
**Pull request by anoukvlug**
In this PR there are some suggestions for the Principles section and the first page of the documentation.

### CODE REVIEW

File: docs/flowlines.rst [hunk 1]
```diff
@@ -83,15 +83,20 @@ to the main flowline (more on this below).
...
 keep for the rest of the workflow). The tail of the tributaries are cut
-according to a distance threshold rule:
+of before reaching the flowline they are tributing to:
 
 .. ipython:: python
 
     @savefig plot_fls_flowlines.png width=80%
     graphics.plot_centerlines(gdir, use_flowlines=True)
 
+This step is needed to avoid sections along the flowline to be accounted for twice.
```
Comment (fmaussion) [inline docs/flowlines.rst:95]:
```suggestion
This step is needed to better represent glacier widths at flowline junctions.
```

File: docs/mass-balance.rst [hunk 2]
```diff
@@ -141,7 +141,7 @@ temperature above which ice melt is assumed to occur (-1°C per default).
 Solid precipitation is computed out of the total precipitation. The fraction of
 solid precipitation is based on the monthly mean temperature: all solid below
 ``temp_all_solid`` (default: 0°C) all liquid above ``temp_all_liq``
-(default: 2°C), linear in between.
+(default: 2°C), linear chancing in between.
```
Comment (fmaussion) [inline docs/mass-balance.rst:144]:
```suggestion
(default: 2°C), linear change in between.
```

**Review by fmaussion — COMMENTED** [review]
Thanks a lot! I made two tiny suggestions which I believe you can accept in the github web interface.

**Comment by anoukvlug** [comment]
Thanks @fmaussion!

**Pull request merged**